pkg/cmd/roachtest/github.go

```diff
@@ -60,10 +60,12 @@ func generateHelpCommand(
 			"How To Investigate (internal)",
 			"https://cockroachlabs.atlassian.net/l/c/SSSBr8c7",
 		)(renderer)
-		issues.HelpCommandAsLink(
-			"Grafana",
-			fmt.Sprintf("https://go.crdb.dev/p/roachfana/%s/%d/%d", clusterName, start.UnixMilli(), end.UnixMilli()),
-		)(renderer)
+		if clusterName != "" {
```

Comment:
Might be worth adding a note that an empty `clusterName` here should correspond to a cluster creation failure.

Reply:
sounds good! will add that